quarto-render/action.yaml

```diff
@@ -0,0 +1,13 @@
+name: "Quarto Render"
+branding:
+  icon: 'award'  
+  color: 'blue'
```

Comment:
Oh I did not knew about that. What is this for ? (just curious)

Reply:
This is just for the blue icon on the [Marketplace page](https://github.com/marketplace/actions/quarto-render). Since we won't be publishing to the Marketplace anymore, though, I'll go ahead and remove this.

Reply:
> This is just for the blue icon on the Marketplace page. 

Oh ok cool !